Frontend/ui-common/src/Animation/Config.purs

```diff
@@ -26,3 +30,36 @@ type AnimConfig = {
   , fromScaleX :: Number
   , toScaleX :: Number
 }
+
+data Direction = LEFT_RIGHT | RIGHT_LEFT | BOTTOM_TOP | TOP_BOTTOM
+
+animConfig :: AnimConfig
+animConfig = {
+    duration : 0
+  , delay : 0
+  , fromY : 0
+  , toY : 0
+  , fromX : 0
+  , toX : 0
+  , repeatCount : (PrestoAnim.Repeat 0)
+  , toAlpha : 0.0
+  , fromAlpha : 0.0
+  , interpolator : PrestoAnim.Bezier 0.37 0.0 0.63 1.0
+  , tag : ""
+  , ifAnim : true
+  , fromRotation : 0
+  , toRotation : 0
+  , fromScaleY : 0.0
+  , toScaleY : 0.0
+  , fromScaleX : 0.0
+  , toScaleX : 0.0
+}
+
+translateYAnimConfig :: Direction -> AnimConfig
+translateYAnimConfig direction = animConfig {
+  duration = 300
```

Comment:
Can duration be smaller? In the video which you shared to me it was 300ms or more?

Reply:
it was 300 only.

Reply:
Yes that's quite slow, can you change it to 150-175ms

Reply:
Done